### PR TITLE
-Add Raspberry Pi profile

### PR DESCRIPTION
**Pull request by chrisknepper**

### CODE REVIEW

File: chapters/raspberrypi.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+- Does your subject participate in social media?
+	- [https://twitter.com/Raspberry_Pi](https://twitter.com/Raspberry_Pi) (205K Followers)
+	- [https://plus.google.com/+raspberrypi/posts](https://plus.google.com/+raspberrypi/posts) (850K Followers)
+	- [https://www.facebook.com/raspberrypi](https://www.facebook.com/raspberrypi) (112K Likes)
+
+### Communications channels for COMPANY
+
+- What communication channels does your subject use to reach their public?
+- Briefly describe and include a URL for each.
+	- [Github](https://github.com/raspberrypi), [educational Github](https://github.com/raspberrypilearning) – Software repos…the former focuses on Pi core software, the latter is comprised of educational projects and demos.
```
Comment (ryansb) [inline chapters/raspberrypi.md]: Can you pull these into footnote links so this list is readable as plaintext?

File: chapters/raspberrypi.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+There are **so** many subjects you could choose to profile, so why did you choose this one?
+What drew you into wanting to know more about the organization?
+How did you/your group decide on and agree?
+
+The Raspberry Pi is a well-known open hardware device, used in a variety of contexts. It helps kids learn programming, is used as the brain of many DIY/physical computing projects, and because it is so cheap, serves to help people get a computer who otherwise might not be able to afford one. The organization responsible for the device is a registered charity.
+
+
+## Organizational Details
+
+- Is the subject of your profile a corporate entity?
```
Comment (ryansb) [inline chapters/raspberrypi.md]: Q&A doesn't make much sense as a nested list. Markdown has blockquote support that would display better. 

> Is the subject of your profile a corporate entity

Yes

> What type?

blah blah blah

Comment (brendanwhitfield) [inline chapters/raspberrypi.md]: Why didn't I think of that... That's *so* much prettier. Opened #30

**Comment by qwertos** [comment]
Directory structure has been updated. Please `git mv` to new structure and I'll merge. :smile:

**Comment by chrisknepper** [comment]
Done! Sorry that took so long.

**Comment by qwertos** [comment]
Please update to new template. :smile:

**Comment by qwertos** [comment]
#114 depreciates and closes this PR. :smile:

**Pull request closed without merging**